luigi_pipeline/seqr_loading.py

```diff
@@ -21,12 +21,17 @@ class SeqrVCFToMTTask(HailMatrixTableTask):
     sample_type = luigi.ChoiceParameter(choices=['WGS', 'WES'], description='Sample type, WGS or WES', var_type=str)
     validate = luigi.BoolParameter(default=True, description='Perform validation on the dataset.')
     dataset_type = luigi.ChoiceParameter(choices=['VARIANTS', 'SV'], default='VARIANTS', description='VARIANTS or SV.')
+    remap_path = luigi.OptionalParameter(default=None, description="Path to a file with two columns: s and seqr_id.")
+    subset_path = luigi.OptionalParameter(default=None, description="Path to a file with sample IDs, s, for subsetting MT.")
```

Comment:
Maybe clarify that these should be TSV files and that the subset file should have sample IDs in a column.